src/Common/src/TypeSystem/Common/TypeSystemConstaintsHelpers.cs

```diff
@@ -2,33 +2,28 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-
-using System.Diagnostics;
-
 namespace Internal.TypeSystem
 {
     public static class TypeSystemConstraintsHelpers
     {
         private static bool VerifyGenericParamConstraint(Instantiation typeInstantiation, Instantiation methodInstantiation, GenericParameterDesc genericParam, TypeDesc instantiationParam)
         {
-            GenericConstraints constraints = genericParam.Constraints;
```

Comment:
Constraints property is virtual call, and the implementation may not be cheap. It was done this way so that it gets called just one.

Reply:
Oh, I see. Sorry, I thought changing it makes the code more readable, but that makes sense.